docs/usage.rst

```diff
@@ -403,11 +403,15 @@ Examples
 borg serve has special support for ssh forced commands (see ``authorized_keys``
 example below): it will detect that you use such a forced command and extract
 the value of the ``--restrict-to-path`` option(s).
+
 It will then parse the original command that came from the client, makes sure
 that it is also ``borg serve`` and enforce path restriction(s) as given by the
 forced command. That way, other options given by the client (like ``--info`` or
 ``--umask``) are preserved (and are not fixed by the forced command).
 
+Environment variables (such as BORG_HOSTNAME_IS_UNIQUE) contained in the original
+command sent by the client are *not* interpreted, but ignored.
```

Comment:
- Add it to /etc/environment
- Add it to ~/.ssh/environment
- Add it to other OS-specific places
- Possibly shrc's and /etc/profile, if a shell indirection is invoked along the way (e.g. `command="some_wrapper_around_borg.sh"`)
- Add it the forced command line, e.g. `command="export BORG_SOMETHING=FOO ; borg serve ...",restrict ssh-ed25519 ...`